randovania/data/json_data/prime1.txt

```diff
@@ -3297,18 +3362,22 @@ Asset id: 3153742515
 > Dock to Fungal Hall B; Heals? False
   * Plasma Door to Fungal Hall B/Dock to Phazon Mining Tunnel
   > Dock to Fungal Hall A
-      All of the following:
-          Power Bomb and Morph Ball
-          Morph Ball Bomb or Boost Ball
+      Morph Ball Bomb and Power Bomb and Morph Ball
   > Pickup (Artifact of Newborn)
-      Morph Ball Bomb and Morph Ball and Phazon Suit
+      All of the following:
+          Morph Ball Bomb and Morph Ball
+          Any of the following:
+              Phazon Damage ≥ 2000
+              Boost Ball and Phazon Damage ≥ 1600
 
 > Dock to Fungal Hall A; Heals? False; Spawn Point
   * Plasma Door to Fungal Hall A/Dock to Phazon Mining Tunnel
   > Dock to Fungal Hall B
       All of the following:
           Power Bomb and Morph Ball
-          Morph Ball Bomb or Boost Ball
+          Any of the following:
+              Morph Ball Bomb
+              Boost Ball and Movement (Beginner)
 
 > Pickup (Artifact of Newborn); Heals? False
   * Pickup 88; Major Location? True
```

Comment:
Related to above, but maybe the damage required without phazon should be split among the two nodes. So reaching without phazon on the previous node needs X depending on the difficulty, then damage required to leave from the pickup node. At least that one will be consistent without phazon suit, so only one lowest level is needed there

Reply:
In my latest commit, I've put all the phazon suit/damage requirements into the FHB door -> item connection and left out any of requirements for the return trip. I didn't want to split up the damage requirements because the non-linear increase of phazon damage is going to make it difficult to figure out the precise damage requirements.

Reply:
I don't think it should be too hard really. regardless of the difficulty except no tricks, the damage for returning would be the same. Could look at 100% runs to get a rough estimate.

Reply:
I've split up the damage based on [this video](https://www.youtube.com/watch?v=zpyhbOerMT4). For the path back, I've added paths for movement levels of 2-4, with a 100 energy decrease at each trick level.

Reply:
Seems good for now. It can always be fixed/adjusted later on as needed.